run-health-checks.sh

```diff
@@ -35,11 +39,8 @@ elif  echo "$SKU" | grep -q "hx176-48rs"; then
 elif  echo "$SKU" | grep -q "hx176-24rs"; then
     conf_name="hx176-24rs"
 else
-    echo "SKU health check currently not implemented"
-    exit 1
+    echo "SKU is currently not supported by Azure health checks." | tee -a $log_path
```

Comment:
Probably worth adding the sku name to the message as long as we're touching the line.  "SKU $SKU..."

Reply:
Let me address

Reply:
addressed